packages/synapse-react-client/src/synapse-queries/KeyFactory.ts

```diff
@@ -44,17 +44,18 @@ const entityQueryKeyObjects = {
   /* Query key for all entities */
   all: { objectType: 'entity' },
   // Data for an entity
-  entity: (id: string) => ({ ...entityQueryKeyObjects.all, id: id }),
-  // Data for a particular version of an entity
-  version: (id: string, versionNumber?: string | number) => ({
-    ...entityQueryKeyObjects.entity(id),
-    versionNumber,
-  }),
+  entity: (id: string, versionNumber?: string | number) => {
+    const key: Record<string, unknown> = {
+      ...entityQueryKeyObjects.all,
+      id: id,
+    }
+    if (versionNumber) {
+      key['version'] = versionNumber
+    }
+    return key
+  },
   // List of Entity Versions
-  versions: (id: string) => ({
-    ...entityQueryKeyObjects.entity(id),
-    scope: 'versions',
```

Comment:
I removed the `scope` property from the key and moved the value to be the next array item. This is effectively the same, I think it's a bit easier to reason with when the entity id/version are kept in a separate object from this label